<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackViz</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/visimpl.png</normaloff>:/visimpl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenBlueConfig"/>
    <addaction name="actionOpenCSVFiles"/>
    <addaction name="actionOpenH5Files"/>
    <addaction name="actionConnectRESTserver"/>
    <addaction name="separator"/>
    <addaction name="actionOpenSubsetEventsFile"/>
    <addaction name="separator"/>
    <addaction name="actionOpenGroupsFile"/>
    <addaction name="separator"/>
    <addaction name="actionCloseData"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionTogglePlaybackDock"/>
    <addaction name="separator"/>
    <addaction name="actionAutoNamingSelections"/>
    <addaction name="actionFill_Plots"/>
    <addaction name="actionShowPanels"/>
    <addaction name="actionAddZeroEQhistograms"/>
    <addaction name="separator"/>
    <addaction name="actionConfigureRESTconnection"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionFocusOnPlayhead"/>
    <addaction name="actionFollowPlayhead"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAdvancedRecorderOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenBlueConfig"/>
   <addaction name="actionOpenCSVFiles"/>
   <addaction name="actionOpenH5Files"/>
   <addaction name="actionConnectRESTserver"/>
   <addaction name="separator"/>
   <addaction name="actionOpenSubsetEventsFile"/>
   <addaction name="separator"/>
   <addaction name="actionTogglePlaybackDock"/>
   <addaction name="actionShowDataManager"/>
   <addaction name="actionShowPanels"/>
   <addaction name="separator"/>
   <addaction name="actionFocusOnPlayhead"/>
   <addaction name="actionFollowPlayhead"/>
   <addaction name="separator"/>
   <addaction name="actionFill_Plots"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionOpenBlueConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-bc.svg</normaloff>:/icons/folder-bc.svg</iconset>
   </property>
   <property name="text">
    <string>Open BlueConfig...</string>
   </property>
   <property name="toolTip">
    <string>Open BlueConfig</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionCloseData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>Close Dataset</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAutoNamingSelections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Naming Selections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFill_Plots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/fill.svg</normaloff>:/icons/fill.svg</iconset>
   </property>
   <property name="text">
    <string>Fill Plots</string>
   </property>
   <property name="toolTip">
    <string>Toggle line/fill plots</string>
   </property>
  </action>
  <action name="actionTogglePlaybackDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/play.svg</normaloff>
     <normalon>:/icons/play.svg</normalon>:/icons/play.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle Playback Dock</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide the playback Dock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShowDataManager">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/generic-info.svg</normaloff>:/icons/generic-info.svg</iconset>
   </property>
   <property name="text">
    <string>Show Data Manager</string>
   </property>
  </action>
  <action name="actionFocusOnPlayhead">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/focus.svg</normaloff>:/icons/focus.svg</iconset>
   </property>
   <property name="text">
    <string>Focus on playhead</string>
   </property>
   <property name="toolTip">
    <string>Focus on playhead</string>
   </property>
  </action>
  <action name="actionFollowPlayhead">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/autofocus.svg</normaloff>:/icons/autofocus.svg</iconset>
   </property>
   <property name="text">
    <string>Follow playhead</string>
   </property>
   <property name="toolTip">
    <string>Follow playhead</string>
   </property>
  </action>
  <action name="actionOpenCSVFiles">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-csv.svg</normaloff>:/icons/folder-csv.svg</iconset>
   </property>
   <property name="text">
    <string>Open CSV Files...</string>
   </property>
   <property name="toolTip">
    <string>Open CSV Files</string>
   </property>
  </action>
  <action name="actionOpenSubsetEventsFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-doc.svg</normaloff>:/icons/folder-doc.svg</iconset>
   </property>
   <property name="text">
    <string>Open Subset Events File...</string>
   </property>
   <property name="toolTip">
    <string>Open subset/events file</string>
   </property>
  </action>
  <action name="actionOpenH5Files">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-h5.svg</normaloff>:/icons/folder-h5.svg</iconset>
   </property>
   <property name="text">
    <string>Open H5 network and activity files...</string>
   </property>
   <property name="toolTip">
    <string>Open HDF5 network and activity files...</string>
   </property>
  </action>
  <action name="actionAddZeroEQhistograms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add ZeroEQ selections as histograms</string>
   </property>
  </action>
  <action name="actionShowPanels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolconfig.svg</normaloff>:/icons/toolconfig.svg</iconset>
   </property>
   <property name="text">
    <string>Show Configuration Panels</string>
   </property>
   <property name="toolTip">
    <string>Shows/Hides the configuration panels.</string>
   </property>
  </action>
  <action name="actionAdvancedRecorderOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advanced recorder options</string>
   </property>
  </action>
  <action name="actionConfigureRESTconnection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Configure REST connection...</string>
   </property>
   <property name="toolTip">
    <string>Configure REST connection options.</string>
   </property>
  </action>
  <action name="actionConnectRESTserver">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-rest.svg</normaloff>:/icons/folder-rest.svg</iconset>
   </property>
   <property name="text">
    <string>Connect to REST server...</string>
   </property>
   <property name="toolTip">
    <string>Connect to a REST API server.</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenGroupsFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-doc.svg</normaloff>:/icons/folder-doc.svg</iconset>
   </property>
   <property name="text">
    <string>Open Groups file...</string>
   </property>
   <property name="toolTip">
    <string>Open groups as histograms</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
